.gitignore

```diff
@@ -5,3 +5,6 @@
 *.html
 urllog.txt
 *log
+.DS_Store
+tox.ini
```

Comment:
Whoops... it wasn't in there before, and I had just added it so my linter would follow the CircleCI settings